README.md

```diff
@@ -41,9 +41,25 @@ dnf install libtool clang-devel bison cmake ninja-build samba flex texinfo glib2
 
 # Basic usage
 
-If you want to start up a QEMU VM running CheriBSD run `cheribuild.py run-riscv64-purecap -d` (-d means build all dependencies).
-If you would like the VM to have all userspace binaries to be built as plain RISC-V binaries instead of CHERI pure-capability ones use `cheribuild.py run-riscv64-hybrid -d`.
-This will build the CHERI compiler, QEMU, CheriBSD, create a disk image and boot that in QEMU.
+In these examples the `cheribuild.py` program is run from within the
+`cheribuild/` directory as `./cheribuild.py` with options following
+the main command.  Please note the leading `./` before the command
+name.  It is also possible to add the `cheribuild/` directory to your
+`PATH` or to link the binary somewhere such as in a `bin/` diretory of
+your choosing.
```

Comment:
Thanks, this new wording looks good to me; anyone who has a clue about Unix will read this and do what they want, and those who don't know what it means can ignore it and blindly follow the basic approach